### PR TITLE
Revert "Revert "Created mBean with OSGi""

### PR DESCRIPTION
**Pull request by FlynnIBM**
This is the same as https://github.com/OpenLiberty/open-liberty/pull/10186. I merged it before I meant to, so I reverted it out.

### CODE REVIEW

**Comment by FlynnIBM** [comment]
#run-libby-bot

**Comment by FlynnIBM** [comment]
#build

File: dev/com.ibm.ws.security.authentication.builtin/src/com/ibm/ws/security/authentication/internal/cache/RevokeAuthCacheUsersImpl.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+import org.osgi.service.component.annotations.ConfigurationPolicy;
+import org.osgi.service.component.annotations.Deactivate;
+import org.osgi.service.component.annotations.Reference;
+import org.osgi.service.component.annotations.ReferencePolicyOption;
+
+import com.ibm.ws.security.authentication.cache.AuthCacheService;
+import com.ibm.ws.security.authentication.cache.RevokeAuthCacheUsers;
+
+/**
+ *
```
Comment (ginnick) [inline dev/com.ibm.ws.security.authentication.builtin/src/com/ibm/ws/security/authentication/internal/cache/RevokeAuthCacheUsersImpl.java]: Can we add a little description here.

**Comment by mmerali96** [comment]
Hi @utle, the name RevokeAuthCacheUsers came from Ajay in a note. We have no preference on what to call it, so it is whatever you or Ajay want it to be.

**Comment by utle** [comment]
> Hi @utle, the name RevokeAuthCacheUsers came from Ajay in a note. We have no preference on what to call it, so it is whatever you or Ajay want it to be.

Please change it, Thanks.

**Comment by mmerali96** [comment]
@utle Changed the name for the mbean and changed the configuration policy to ignore as per your comments.

**Comment by mmerali96** [comment]
#build

**Comment by mmerali96** [comment]
#run-libby-bot

**Pull request merged**